src/providers/ipa/ipa_views.c

```diff
@@ -27,6 +27,118 @@
 #include "util/cert.h"
 #include "providers/ldap/sdap_async.h"
 #include "providers/ipa/ipa_id.h"
+#include "db/sysdb.h"
+
+enum is_user_or_group {
+    IS_USER,
+    IS_GROUP
+};
```

Comment:
Hi,

there is already `enum sysdb_obj_type` maybe you can reuse this?

bye,
Sumit

Reply:
Fixed. Thank you.